GenderPayGap.Database/Migrations/20200504111013_InitialCreate.cs

```diff
@@ -873,102 +879,94 @@ protected override void Up(MigrationBuilder migrationBuilder)
 
             migrationBuilder.CreateIndex(
-                name: "IX_SicSectionId",
+                name: "IX_SicCodes_SicSectionId",
                 table: "SicCodes",
                 column: "SicSectionId");
 
             migrationBuilder.CreateIndex(
-                name: "IX_AddressId",
+                name: "IX_UserOrganisations_AddressId",
                 table: "UserOrganisations",
                 column: "AddressId");
 
             migrationBuilder.CreateIndex(
-                name: "IX_OrganisationId",
+                name: "IX_UserOrganisations_OrganisationId",
                 table: "UserOrganisations",
                 column: "OrganisationId");
 
             migrationBuilder.CreateIndex(
-                name: "IX_UserId",
+                name: "IX_UserOrganisations_UserId",
                 table: "UserOrganisations",
                 column: "UserId");
 
             migrationBuilder.CreateIndex(
-                name: "IX_ContactEmailAddress",
+                name: "IX_Users_ContactEmailAddress",
                 table: "Users",
                 column: "ContactEmailAddress");
 
             migrationBuilder.CreateIndex(
-                name: "IX_ContactPhoneNumber",
+                name: "IX_Users_ContactPhoneNumber",
                 table: "Users",
                 column: "ContactPhoneNumber");
 
             migrationBuilder.CreateIndex(
-                name: "IX_EmailAddress",
+                name: "IX_Users_EmailAddress",
                 table: "Users",
                 column: "EmailAddress");
 
             migrationBuilder.CreateIndex(
-                name: "IX_StatusId",
+                name: "IX_Users_StatusId",
                 table: "Users",
                 column: "StatusId");
 
             migrationBuilder.CreateIndex(
-                name: "IX_UserId",
+                name: "IX_UserSettings_UserId",
                 table: "UserSettings",
                 column: "UserId");
 
             migrationBuilder.CreateIndex(
-                name: "IX_ByUserId",
+                name: "IX_UserStatus_ByUserId",
                 table: "UserStatus",
                 column: "ByUserId");
 
             migrationBuilder.CreateIndex(
-                name: "IX_StatusDate",
+                name: "IX_UserStatus_StatusDate",
                 table: "UserStatus",
                 column: "StatusDate");
 
             migrationBuilder.CreateIndex(
-                name: "IX_UserId",
+                name: "IX_UserStatus_UserId",
                 table: "UserStatus",
                 column: "UserId");
-            
-            migrationBuilder.AddForeignKey(
-                name: "FK_OrganisationPublicSectorTypes_Organisations_OrganisationId",
-                table: "OrganisationPublicSectorTypes",
-                column: "OrganisationId",
-                principalTable: "Organisations",
-                principalColumn: "OrganisationId",
-                onDelete: ReferentialAction.Cascade);
```

Comment:
This looks surprising.
I think there should still be a foreign key between `Organisation.OrganisationId` and `OrganisationPublicSectorTypes.OrganisationId`.

I know this is an auto-generated file, so you didn't deliberately make this change.
I can't actually see what would have caused this change to occur!
But I think we should try to make it so this foreign key still exists.

Reply:
I think I added this manually when I squashed the migrations. It was automatically removed then too, so I had to add it back in to make the schemas match. I'll add it back in again here too.